dotcom-rendering/scripts/webpack/server.js

```diff
@@ -1,7 +1,20 @@
+const FriendlyErrorsWebpackPlugin = require('friendly-errors-webpack-plugin');
+const chalk = require('chalk');
 const GuStatsReportPlugin = require('./gu-stats-report-plugin');
 
 const DEV = process.env.NODE_ENV === 'development';
 
+const friendlyErrorsWebpackPlugin = () =>
```

Comment:
What's the point of this abstraction? I know we do it in `browser.js` - but I can't see the reason for it existing there either is. Could be bad for maintenance as someone new to the code would wonder the same thing, I assumed it was used elsewhere, but it's not.

Reply:
Removed!